### PR TITLE
Remove outdated parts of the roadmap

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

**Comment by laurentlb** [comment]
cc @sventiffe @jhfield

File: roadmap.md [hunk 1]
```diff
@@ -5,56 +5,10 @@ title: Roadmap
...
-    _gradual adoption_ of Bazel.
-*   [**Configurability**](roadmaps/configuration.html). Outside of a few
-    hardcoded cases (fat apk for Android,host -> target transitions), the promise
-    of single-command multi-platform builds is unfulfilled. 
-*   [**Starlark API and tooling**](roadmaps/starlark.html). We need to support the rule authors with
-    **clear, documented, stable API**, and **strong language and tooling**:
-    documentation generators, refactoring tools, testing frameworks and so on.
-
 ## Roadmaps for specific Bazel areas
 *  [Android](https://github.com/bazelbuild/rules_android/blob/master/ROADMAP.md)
```
Comment (jin) [inline roadmap.md:9]: Android is also outdated.

Comment (laurentlb) [reply roadmap.md:9]: @timpeut - any update on this?

(I skipped the annotation here because it's separate repository.)

**Comment by laurentlb** [comment]
@timpeut: what shall we do with the Android roadmap?

@sventiffe @jhfield @meteorcloudy: are you okay with the change?

**Comment by laurentlb** [comment]
I've removed the Android, C++, and Coverage roadmaps as they are outdated.

I think the PR is ready now.

**Pull request merged**